lib/database.js

```diff
@@ -54,6 +54,7 @@ var CrudDriver = function(opts) {
     insert: insert,
     update: update,
     delete: deleteItem,
+    procedure: opts.procedure,
```

Comment:
If someone attempted to create this using the MySql driver, you should get a not implemented error raised